src/Factory/RestControllerFactory.php

```diff
@@ -143,7 +143,6 @@ public function createServiceWithName(ServiceLocatorInterface $controllers, $nam
             ));
         }
 
-        $controller->setEventManager($events);
```

Comment:
Just feels weird that the app is broken after an update. But yes, okay, you got me ;-) I did not read that part of the documentation thoroughly enough.